### PR TITLE
Update to latest VSC API

### PR DESCRIPTION
**Pull request by DonJayamanne**
Updates to latest API

### CODE REVIEW

File: src/client/index.tsx [hunk 1]
```diff
@@ -16,53 +16,56 @@ __webpack_public_path__ = getPublicPath();
...
-            tag.parentNode?.insertBefore(container, tag.nextSibling); // NOSONAR
-        }
-        tag.parentElement?.removeChild(tag); // NOSONAR
+        console.error('request', request);
+        const output = convertVSCodeOutputToExecutResultOrDisplayData(request);
+        console.log(`Rendering mimeType ${request.mimeType}`, output);
+        console.error('request output', output);
 
-        ReactDOM.render(React.createElement(CellOutput, { mimeType, output }, null), container);
+        ReactDOM.render(React.createElement(CellOutput, { mimeType: request.mimeType, output }, null), request.element);
```
Comment (rchiodo) [inline src/client/index.tsx:38]: Oh no cellOutput is still used. We need the changes made to our rendering code ported as well.

Comment (rchiodo) [reply src/client/index.tsx:38]: This file here probably needs some changes to get escaping to work correctly. I'll add an issue

Comment (rchiodo) [reply src/client/index.tsx:38]: https://github.com/microsoft/vscode-notebook-renderers/blob/master/src/client/render.tsx

Comment (DonJayamanne) [reply src/client/index.tsx:38]: I'll create a seprate issue for that.

Comment (DonJayamanne) [reply src/client/index.tsx:38]: @rchiodo I have added an issue for this & made a reference to the original PR you submitted.

**Review by IanMatthewHuff — APPROVED** [review]

**Review by DavidKutu — APPROVED** [review]

**Review by joyceerhl — APPROVED** [review]

**Pull request merged**